blog/models.py

```diff
@@ -3,25 +3,42 @@
 
 class Blog(models.Model):
     title = models.CharField(max_length=100, blank=True, default='')
-    description = models.TextField(max_length=1500)
+    description = models.TextField(max_length=50000)
     created = models.DateTimeField(auto_now_add=True)
     owner = models.ForeignKey('auth.User', related_name='blog', on_delete=models.CASCADE, null=True)
     image = models.FileField(blank=True, null=True)
 
+    @property
+    def votes(self):
+        upvotes = UserVote.objects.filter(blog=self, vote_type='up').count()
+        downvotes = UserVote.objects.filter(blog=self, vote_type='down').count()
```

Comment:
Try to combine these 2 queries in order to get the calculated votes. Also declare the class variables for up vote type and down vote type and use them in the project.

Reply:
I looked into Annotation/Aggregation but I can't find a better solution.

Although I've added the VOTE_CHOICE in the Vote Model.

Reply:
I've done this using aggregate.
I'm updating the PR.